### PR TITLE
:recycle: Use patch instead of sed, Fix #149

### LINKED ISSUE #149: Window icon no longer works under KDE Wayland
Until earlier today, vscodium windows had a proper icon in my KDE Wayland Plasma panel taskbar. Today I updated my flatpaks and system packages and did a reboot, and now the icon is broken. The vscodium package got updated to 1.73.1.22314, so I suspect that is where the problem started -- though I don't know how to figure out which version I had installed before.

According to kwin, the `desktopFileName` property of vscodium windows is now set to 'codium-url-handler', which obviously is not correct. (I don't know if before this update, the app has been using Wayland or X11.)

### PR DESCRIPTION
**Pull request by noonsleeper**

### CODE REVIEW

File: patches/product.json.patch [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@@ -291,7 +291,7 @@
+   },
+   "commit": "9f4ee35641a60d86b7ba9e567ed46b712b80ae37",
+   "date": "2024-06-13T18:53:22.307Z",
+-  "checksums": {
++  "_checksums": {
```
Comment (daiyam) [inline patches/product.json.patch:6]: @noonsleeper Can you avoid to disable the checksums checks?

Comment (noonsleeper) [reply patches/product.json.patch:6]: Yes, I don't remember why we disable that, maybe was for the IPC communication between instances, I will test removing that line

Comment (noonsleeper) [reply patches/product.json.patch:6]: @daiyam, I did a test and is not needed, I think, then I push a fix for this

Comment (daiyam) [reply patches/product.json.patch:6]: Thx

**Pull request merged**